storage/acl/acl_test.go

```diff
@@ -27,60 +29,81 @@ import (
 // TestACL runs all of the package tests.
 func TestACL(t *testing.T) {
 	tc := testutil.SystemTest(t)
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatalf("storage.NewClient: %v", err)
+	}
+	defer client.Close()
+
 	var (
 		bucket                = tc.ProjectID + "-samples-object-bucket-1"
 		object                = "foo.txt"
-		buf                   bytes.Buffer
 		allAuthenticatedUsers = storage.AllAuthenticatedUsers
-		roleReader            = storage.RoleReader
 	)
 
-	if err := addBucketACL(bucket, allAuthenticatedUsers, roleReader); err != nil {
-		t.Errorf("addBucketACL: %v", err)
+	b := client.Bucket(bucket)
+	if err := b.Create(ctx, tc.ProjectID, nil); err != nil {
+		t.Fatalf("Bucket(%q).Create: %v", bucket, err)
 	}
-	if err := addDefaultBucketACL(bucket, allAuthenticatedUsers, roleReader); err != nil {
-		t.Errorf("addDefaultBucketACL: %v", err)
+
+	// Open local file.
+	f, err := os.Open("notes.txt")
```

Comment:
I think we can just write directly. No need to read from a file. `fmt.Fprint(w, "Hello")`

Reply:
Good idea, pushed!